trackingC.Rmd

```diff
@@ -567,6 +567,28 @@ ggplot(plot_data, aes(x = year, fill = def)) +
 
 ```
 
+``` {r af-trend, eval=FALSE}
+trend <- plot_data %>%
+  filter(year %in% 1960:2020)
+
+af_trend <-  trend %>%
+  group_by(ssp, def) %>%
```

Comment:
This is fine, but it's why the error values are so small—we're computing based on the single, already-summarized SSP/definition numbers. I think that 'real' errors would reflect the larger diversity among individual runs.

I'd leave for now, though.